src/pages/Admin.jsx

```diff
@@ -14,81 +15,87 @@ const Admin = () => {
         alignItems: "center",
       }}
     >
-      <Button
-        sx={{
-          fontSize: "18px",
-          height: "150px",
-          width: "340px",
-          padding: "64px 32px",
-          textTransform: "none",
-          bgcolor: "white",
-          transition: "all 0s ease-in-out",
-          color: "#000",
-          "&:hover": {
-            bgcolor: "primary.main",
-            color: "white",
-            height: "160px",
-            width: "350px",
-            fontSize: "20px",
-            padding: "69px 37px",
-          },
-        }}
-        size="large"
-        variant="contained"
-        startIcon={<Groups2 />}
-      >
-        Employee Management
-      </Button>
-      <Button
-        sx={{
-          height: "150px",
-          width: "340px",
-          padding: "64px 32px",
-          textTransform: "none",
-          bgcolor: "#fff",
-          fontSize: "18px",
-          transition: "all 0s ease-in-out",
-          color: "#000",
-          "&:hover": {
-            bgcolor: "primary.main",
-            color: "white",
-            fontSize: "20px",
-            height: "160px",
-            width: "350px",
-            padding: "69px 37px",
-          },
-        }}
-        size="large"
-        variant="contained"
-        startIcon={<AirlineSeatFlatAngledTwoTone />}
-      >
-        Patients Management
-      </Button>
-      <Button
-        sx={{
-          fontSize: "18px",
-          textTransform: "none",
-          height: "150px",
-          width: "340px",
-          padding: "64px 32px",
-          bgcolor: "white",
-          transition: "all 0s ease-in-out",
-          color: "#000",
-          "&:hover": {
-            bgcolor: "primary.main",
-            color: "white",
-            padding: "69px 37px",
-            fontSize: "20px",
-            height: "160px",
-            width: "350px",
-          },
-        }}
-        size="large"
-        variant="contained"
-        startIcon={<AirlineSeatFlatAngledTwoTone />}
-      >
-        In-Service Management
-      </Button>
+      <Link to="/employees">
+        <Button
```

Comment:
I think it is more efficient to use the Link inside the Button rather that outside the Button . something like       

<Button
        component={Link}
        to="/employees"